test/fixtures/mson-inheritance.json

```diff
@@ -12,11 +12,15 @@
         {
           "element": "object",
           "meta": {
-            "id": "User"
+            "id": "User",
+            "description": "The User description\n\n"
```

Comment:
It is copy of "raw" text in form as it come from parser without any changes.
In general we can add some kind of "filters"